pebblo/app/pebblo-ui/src/util.js

```diff
@@ -15,18 +13,44 @@ const MONTHS = [
   "Dec",
 ];
 
-export const get_Formatted_Date = (date) => {
-  const newDate = new Date(date);
-  return `${newDate.getDate()} ${MONTHS[newDate.getMonth()]
-    } ${newDate.getFullYear()}`;
+export const getFormattedDate = (
+  date,
+  showTime = false,
+  showTimeZone = false
+) => {
+  if (date) {
+    const newDate = new Date(date);
+    const dateOptions = { day: "2-digit", month: "short", year: "numeric" };
+    if (showTime) {
+      dateOptions.hour = "numeric";
+      dateOptions.minute = "numeric";
+    }
+    if (showTimeZone) {
+      dateOptions.timeZoneName = "long";
```

Comment:
In the pdf report, we do not have long timezone format. Is it explicitly kept long on Local UI?

Reply:
We chose to use the long formats here because some of the short forms can be similar. I'll go ahead and update it for report pdfs as well.